admin/hardware-inventory.md

```diff
@@ -4,9 +4,9 @@
 
 | Item | Units | Status |
 |:-----|:------|:-------|
-| Raspberry Pi 3B+ with power | 6 | With @benhylau |
-| Toplinkst TOP-S5 | 45 | Waiting for USB adapters |
-| Samsung Evo Select 32 GB | 20 | With @benhylau |
+| Raspberry Pi 3B+ with power | 6 | With @darkdrgn2k |
+| Toplinkst TOP-S5 | 45 | With @darkdrgn2k |
```

Comment:
This doesn't mean that @darkdrgn2k has 45 TOP-S5s on hand, correct?
Now that @makeworld-the-better-one and @benhylau each got one of of these, I expected this number to go down by two, but I'm probably not reading this right.